packages/fiat-p256/fiat-p256.0.2.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer: "Etienne Millon <me@emillon.org>"
+license: "MIT"
+authors: [
+    "Etienne Millon <me@emillon.org>"
+    "Andres Erbsen <andreser@mit.edu>"
+    "Google Inc."
+    "Jade Philipoom <jadep@mit.edu> <jade.philipoom@gmail.com>"
+    "Massachusetts Institute of Technology"
+]
+homepage: "https://github.com/mirage/fiat"
+bug-reports: "https://github.com/mirage/fiat/issues"
+dev-repo: "git+https://github.com/mirage/fiat.git"
+doc: "https://mirage.github.io/fiat/doc"
+tags: ["org:mirage"]
+build: [
+ ["dune" "subst"] {pinned}
+ ["dune" "build" "-p" name "-j" jobs]
+ ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "alcotest" {with-test}
+  "asn1-combinators" {with-test}
+  "benchmark" {with-test}
+  "bigarray-compat"
+  "cstruct" {>= "3.5.0"}
+  "dune" {build & >= "1.6.0"}
```

Comment:
```suggestion
  "dune" {>= "1.6.0"}
```
Dune should not be tagged as a build dependency anymore (see https://github.com/ocaml/opam-repository/pull/14266)

Reply:
cool, good to see it's policy now!